### DISCUSSION TITLE
How to exit insert mode with "jk" ?

### DISCUSSION THREAD

**Discussion by 0xGutz** [LunarVim/LunarVim#3935] [Q&A] [answered]
### Problem description

I recently installed lvim but I'm not sure if I should edit the config.lua inside my ~/.config directory... Or the init.lua file inside the local/share/lunarvim directory. I tried editing the latter but wasn't sure what the correct line to do this? 

Would it be `["<jk>"] = "<Esc>"`
Or
`lvim.keys.insert_mode["<jk>"] = "<Esc>"`

I've tried both but neither seems to work...

### LunarVim version

master-a7e7f286-dirty

### Neovim version (>= 0.8.0)

NVIM v0.9.0-dev

### Operating system/version

Ubuntu 22.04

**Comment by sharkby7e** [comment, marked as answer]
<> are reserved for combining with modifiers. 
use:
``` 
lvim.keys.insert_mode["jk"] = "<ESC>"
```
instead! Worked for me!
